testkit/mocksessionmanager.go

```diff
@@ -26,6 +29,9 @@ type MockSessionManager struct {
 	PS      []*util.ProcessInfo
 	SerID   uint64
 	TxnInfo []*txninfo.TxnInfo
+	Dom     *domain.Domain
```

Comment:
Why do you need this? To get the domain from a session, you could use `domain.Get(GetStore())`